pkg/controller/stacks/stack/stack_test.go

```diff
@@ -428,8 +447,40 @@ func TestCreate(t *testing.T) {
 		name       string
 		r          *v1alpha1.Stack
 		clientFunc func(*v1alpha1.Stack) client.Client
+		allowCore  bool
 		want       want
 	}{
+		{
+			name: "FailRestricedPermissions",
```

Comment:
nit typo: `FailRestricedPermissions` => `FailRestrictedPermissions`

Reply:
@jbw976 will fix and also update design doc :) Manual testing results coming shortly!